orchestration/cleanup_agents.sh

```
@@ -0,0 +1,71 @@
+
+echo "🧹 Orchestration Agent Cleanup Script"
+echo "====================================="
+
+# Configuration
+RESULT_DIR="/tmp/orchestration_results"
+MAX_AGE_HOURS=24
+
+# Function to check if worktree is safe to remove
+is_worktree_safe_to_remove() {
+    local worktree_path="$1"
+    local branch_name=$(git -C "$worktree_path" branch --show-current 2>/dev/null || echo "")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)